packages/contracts/src/contracts/current/tokens/ERC20Token/ERC20Token.sol

```diff
@@ -26,7 +27,14 @@ contract ERC20Token is IERC20Token {
         public
         returns (bool)
     {
-        require(balances[msg.sender] >= _value && balances[_to] + _value >= balances[_to]);
+        require(
+            balances[msg.sender] >= _value,
+            "Insufficient balance to complete transfer."
```

Comment:
This should be a constant since this identical error message is used in two places.